### ISSUE TITLE
Mac OSX Mojave Crash Visual Studio Error Report Points to Electron

### ISSUE THREAD

**Issue by zrodriguez89** [electron/electron#22902]
### Preflight Checklist

* [x] I have read the [Contributing Guidelines](https://github.com/electron/electron/blob/master/CONTRIBUTING.md) for this project.
* [x] I agree to follow the [Code of Conduct](https://github.com/electron/electron/blob/master/CODE_OF_CONDUCT.md) that this project adheres to.
* [x] I have searched the issue tracker for an issue that matches the one I want to file, without success.

Previous issues have been posted to VSCode GitHub. They encouraged people to post here, but based on searching user profiles, they never did. Links:

This may be a duplicate of [Crash on macOS with Electron v3.1.6](https://github.com/electron/electron/issues/18262)

But I don't know if the problem is the same as what they supposedly narrowed it down to. Plus they never followed up with more testing. 

03/2020- [vscode v 1.41.1 crashing on Mac OSX Catalina](https://github.com/microsoft/vscode/issues/88500)

04/2019-[crash when closing vscode in macOS(10.14.5)](https://github.com/microsoft/vscode/issues/77568)

02/9/2019- [VSCode doesn't boot properly on vscode 1.31](https://github.com/microsoft/vscode/issues/68307)

In the case of Catalina:

**original issue beta-** https://github.com/microsoft/vscode/issues/80236

**Apparent problem 08/2019-** https://apple.stackexchange.com/questions/367283/apps-that-are-chromium-or-electron-base-crashes-on-catalina-beta-6-19a536g

### Issue Details

[Crash Reports Gist](https://gist.github.com/zrodriguez89/098827f860663f27dc0a9612daf1d9da)

* **Electron Version:**

Where can I find electron version corresponding to VSCode?

Both errors on Mac OS X 10.14.6 - Mojave
Both errors VSCode version 1.43.0

### Expected Behavior

Use VSCode regularly without crashing

### Actual Behavior

Freezes up then crashes.

### To Reproduce

The best way to reproduce is using VSCode to run SQL code while using the PostgreSQL Microsoft extension. Also need to install pgadmin (I use pgcli). I can also provide SQL files to setup and use.

### Screenshots

None as of now

### Additional Information

I have had it crash on the newest VSCode version, but currently don't have a saved bug report. I can try to get one if needed.

**Comment by sofianguy** [comment]
@zrodriguez89 Thank you for your issue! A lot of this seems very specific to VS Code. Can you file an issue with them directly? https://github.com/microsoft/vscode/issues/new/choose

**Comment by deepak1556** [comment]
Yes please file issue with vscode first and then once the cause is narrowed down to electron, an issue should be filed here with minimal repro. Thanks!

For this specific issue, here is the symbolicated trace.

```
content::BrowserPluginGuest::OnShowPopup(content::RenderFrameHost*, FrameHostMsg_ShowPopup_Params const&) (in Electron Framework) (browser_plugin_guest.cc:1094)
electron::WebViewGuestDelegate::OnZoomLevelChanged(content::WebContents*, double, bool) (in Electron Framework) (web_view_guest_delegate.cc:82)
content::BrowserPluginGuest::OnShowPopup(content::RenderFrameHost*, FrameHostMsg_ShowPopup_Params const&) (in Electron Framework) (browser_plugin_guest.cc:0)
content::WebContentsViewChildFrame::GetNativeView() const (in Electron Framework) (web_contents_view_child_frame.cc:47)
base::internal::Invoker<base::internal::BindState<base::internal::IgnoreResultHelper<void (content::RenderWidgetHostViewMac::*)(cc::RenderFrameMetadata const&)>, base::WeakPtr<content::RenderWidgetHostViewMac>, cc::RenderFrameMetadata>, void ()>::RunOnce(base::internal::BindStateBase*) (in Electron Framework) (bind_internal.h:634)
base::internal::Invoker<base::internal::BindState<void (download::DownloadDestinationObserver::*)(long long, std::__1::unique_ptr<crypto::SecureHash, std::__1::default_delete<crypto::SecureHash> >), base::WeakPtr<download::DownloadDestinationObserver>, long long, std::__1::unique_ptr<crypto::SecureHash, std::__1::default_delete<crypto::SecureHash> > >, void ()>::RunOnce(base::internal::BindStateBase*) (in Electron Framework) (bind_internal.h:634)
vp8_tokenize_mb (in Electron Framework) (tokenize.c:379)
vp8_update_zbin_extra (in Electron Framework) (vp8_quantize.c:425)
point_double (in Electron Framework) (p256.c:234)
BN_mod_sqrt (in Electron Framework) (sqrt.c:248)
ec_GFp_mont_add (in Electron Framework) (ec_montgomery.c:323)
ec_GFp_mont_dbl (in Electron Framework) (ec_montgomery.c:345)
SIKE_keypair (in Electron Framework) (sike.c:427)
std::__1::pair<std::__1::__wrap_iter<std::__1::pair<std::__1::basic_string<char, std::__1::char_traits<char>, std::__1::allocator<char> >, std::__1::unique_ptr<base::Value, std::__1::default_delete<base::Value> > >*>, bool> base::internal::flat_tree<std::__1::basic_string<char, std::__1::char_traits<char>, std::__1::allocator<char> >, std::__1::pair<std::__1::basic_string<char, std::__1::char_traits<char>, std::__1::allocator<char> >, std::__1::unique_ptr<base::Value, std::__1::default_delete<base::Value> > >, base::internal::GetKeyFromValuePairFirst<std::__1::basic_string<char, std::__1::char_traits<char>, std::__1::allocator<char> >, std::__1::unique_ptr<base::Value, std::__1::default_delete<base::Value> > >, std::__1::less<void> >::emplace_key_args<std::__1::basic_string<char, std::__1::char_traits<char>, std::__1::allocator<char> >, std::__1::piecewise_construct_t const&, std::__1::tuple<std::__1::basic_string<char, std::__1::char_traits<char>, std::__1::allocator<char> > const&>, std::__1::tuple<std::__1::unique_ptr<base::Value, std::__1::default_delete<base::Value> >&&> >(std::__1::basic_string<char, std::__1::char_traits<char>, std::__1::allocator<char> > const&, std::__1::piecewise_construct_t const&, std::__1::tuple<std::__1::basic_string<char, std::__1::char_traits<char>, std::__1::allocator<char> > const&>&&, std::__1::tuple<std::__1::unique_ptr<base::Value, std::__1::default_delete<base::Value> >&&>&&) (in Electron Framework) (flat_tree.h:961)
SIKE_keypair (in Electron Framework) (sike.c:427)
__CFRUNLOOP_IS_CALLING_OUT_TO_A_SOURCE0_PERFORM_FUNCTION__ + 17
__CFRunLoopDoSource0 + 108
__CFRunLoopDoSources0 + 195
__CFRunLoopRun + 1189
CFRunLoopRunSpecific + 455
RunCurrentEventLoopInMode + 292
ReceiveNextEventCommon + 603
_BlockUntilNextEventMatchingListInModeWithFilter + 64
_DPSNextEvent + 1135
-[NSApplication(NSEvent) _nextEventMatchingEventMask:untilDate:inMode:dequeue:] + 1361
-[NSApplication run] + 699
SIKE_encaps (in Electron Framework) (sike.c:0)
sike_get_A (in Electron Framework) (isogeny.c:201)
ec_GFp_mont_dbl (in Electron Framework) (ec_montgomery.c:347)
ed25519_set_priv_raw (in Electron Framework) (p_ed25519_asn1.c:31)
bool IPC::MessageT<BrowserPluginHostMsg_UnlockMouse_ACK_Meta, std::__1::tuple<int>, void>::Dispatch<content::BrowserPluginGuest, content::BrowserPluginGuest, void, void (content::BrowserPluginGuest::*)(int)>(IPC::Message const*, content::BrowserPluginGuest*, content::BrowserPluginGuest*, void*, void (content::BrowserPluginGuest::*)(int)) (in Electron Framework) (ipc_message_templates.h:143)
content::BrowserPluginGuest::OnMessageReceived(IPC::Message const&) (in Electron Framework) (browser_plugin_guest.cc:760)
content::BrowserPluginEmbedder::AreAnyGuestsCurrentlyAudible() (in Electron Framework) (browser_plugin_embedder.cc:205)
content::protocol::Browser::DispatcherImpl::setPermission(int, std::__1::basic_string<char, std::__1::char_traits<char>, std::__1::allocator<char> > const&, std::__1::basic_string<char, std::__1::char_traits<char>, std::__1::allocator<char> > const&, std::__1::unique_ptr<content::protocol::DictionaryValue, std::__1::default_delete<content::protocol::DictionaryValue> >, content::protocol::ErrorSupport*) (in Electron Framework) (browser.cc:296)
content::protocol::Browser::DispatcherImpl::setPermission(int, std::__1::basic_string<char, std::__1::char_traits<char>, std::__1::allocator<char> > const&, std::__1::basic_string<char, std::__1::char_traits<char>, std::__1::allocator<char> > const&, std::__1::unique_ptr<content::protocol::DictionaryValue, std::__1::default_delete<content::protocol::DictionaryValue> >, content::protocol::ErrorSupport*) (in Electron Framework) (browser.cc:266)
void std::__1::vector<std::__1::unique_ptr<ui_devtools::protocol::CSS::Value, std::__1::default_delete<ui_devtools::protocol::CSS::Value> >, std::__1::allocator<std::__1::unique_ptr<ui_devtools::protocol::CSS::Value, std::__1::default_delete<ui_devtools::protocol::CSS::Value> > > >::__emplace_back_slow_path<std::__1::unique_ptr<ui_devtools::protocol::CSS::Value, std::__1::default_delete<ui_devtools::protocol::CSS::Value> > >(std::__1::unique_ptr<ui_devtools::protocol::CSS::Value, std::__1::default_delete<ui_devtools::protocol::CSS::Value> >&&) (in Electron Framework) (vector:1677)
content::protocol::Browser::PermissionDescriptor::fromValue(content::protocol::Value*, content::protocol::ErrorSupport*) (in Electron Framework) (browser.cc:75)
AtomMain + 84
0x000009e0 (in Electron Framework)
start + 1
```

This is fixed in vscode latest stable release `1.43.2` , please update to it.
